test/test_Datawizard.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from aperturedb.Blobs import Blobs
+from aperturedb.BoundingBoxes import BoundingBoxes
+from aperturedb.Entities import Entities
+from aperturedb.Images import Images
+from aperturedb.Query import EntityType, Query
+from aperturedb.Utils import Utils
+
+logger = logging.getLogger(__file__)
+
+
+class TestDataWizardEntities():
+    def test_get_persons(self, insert_data_from_csv, db):
+        loaded = insert_data_from_csv(in_csv_file="./input/persons.adb.csv")
+        all_persons = Entities.retrieve(db,
+                                        spec=Query.spec(with_class=EntityType.CUSTOM, custom_class_name="Person"))[0]
+        assert len(all_persons) == len(loaded)
+        too_young = all_persons.filter(lambda p: p["age"] < 18)
+        too_old = all_persons.filter(lambda p: p["age"] > 60)
+        working_people = all_persons - too_old - too_young
+
+        assert all([p["age"] >= 18 and p["age"] <= 60 for p in working_people])
+        ordered = working_people.sort(key=lambda p: p["age"])
+        youngest_working_five = ordered[:5]
+        eldest_working_five = ordered[-5:]
+
+        assert len(youngest_working_five) == 5
+        assert len(eldest_working_five) == 5
+        assert youngest_working_five[0]["age"] < eldest_working_five[0]["age"]
+
+    def test_get_persons_constraints(self, retired_persons):
+        logger.info(f"Retired persons count = {len(retired_persons)}")
+        assert all(p["age"] >= 60 for p in retired_persons)
+
+    def test_get_persons_display(self, retired_persons):
+        logger.info(f"Retired persons count = {len(retired_persons)}")
+        df = retired_persons.inspect()
+        logger.info(f"\n{df}")
+        ages = df['age']
+        assert ages[ages >= 60].count() == len(df)
+
+    def test_update_properties(self, retired_persons, utils: Utils):
+        risk_factors = [{
+            "risk_factor": (p["age"] - 60 // 10)
+        } for p in retired_persons]
+        retired_persons.update_properties(risk_factors)
+        df = retired_persons.inspect()
```

Comment:
there is no testing here, just printing.

Reply:
will do.